### ISSUE TITLE
Adding folder to resource unpacks contents

### ISSUE THREAD

**Issue by Castronova** [hydroshare/hydroshare#2901]
1. Create Resource
2. Drag and drop a folder with data into content

Expected:
- files are uploaded with folder structure intact
e.g.
```
dat
 - cuahsi-members.csv
 - university-data.csv
 - collect_universities.py
```
Actual:
- Data is unpacked from folder and uploaded to the root directory

![image](https://user-images.githubusercontent.com/4822372/45572581-9b1ed000-b837-11e8-92ad-a672a0f4ccc4.png)

**Comment by Castronova** [comment]
This is more problematic for directories with commonly named files. For example uploading the following directory

![image](https://user-images.githubusercontent.com/4822372/45632154-a6a60d00-ba6a-11e8-9423-8d4c4cf27faa.png)

Produces the following error because HS is unpacking the directories.

![image](https://user-images.githubusercontent.com/4822372/45632122-9a21b480-ba6a-11e8-9541-c58659c2e73f.png)

**Comment by dtarb** [comment]
@Castronova this is fixed in the code slated for 1.16 release.  You can check the deployment on https://dev-hs-6.cuahsi.org/.  Consider closing now - or when you test this on www whenever this deploys.

**Comment by aphelionz** [comment]
Closing this - please re-open if it re-appears while testing 1.16
